server.js

```diff
@@ -17,8 +17,13 @@ const execAll = async () => {
 	app.emit('scrapper_finished');
 };
 
+const execNyt = () => scraper.nytData(keys, redis);
+
 execAll();
+execNyt();
+
 setInterval(execAll, config.interval);
+setInterval(execNyt, config.nyt_interval);
```

Comment:
would need Ravy to update config.json - talk to her on this

Reply:
we would need to anway since my last pr changed the redis keys, or not?